docs/guides/organizations/get-started/quickstart.astro.mdx

```diff
@@ -0,0 +1,300 @@
+---
+title: Get started with Organizations and Astro
+description: Create and manage organizations in your Astro app in this guided tutorial.
+sdk: astro
+---
+
+<TutorialHero
+  beforeYouStart={[
+    {
+      title: "Set up a Clerk + Astro application",
+      link: "/docs/astro/getting-started/quickstart",
+      icon: "clerk",
+    }
+  ]}
+/>
+
```

Comment:
<img width="695" height="350" alt="Image" src="https://github.com/user-attachments/assets/dfe54056-6b1b-4a19-a6db-b1bf83fab882" />

there's a huge gap between these two for some reason 🤔